installer/pyinstaller/build-linux.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 set -eu
 
-yum install -y zlib-devel openssl-devel
+yum install -y zlib-devel openssl-devel libffi-devel
```

Comment:
Do we still need `libffi-devel` installation + `ldconfig` run below? I thought these might break the native executable build step?

Reply:
For our own installer pipeline, because we already have `libffi-devel` in the image, this will not do anything and ldd output is exactly the same as before.